mrblib/mruby-cli/setup.rb

```diff
@@ -386,11 +393,43 @@ def clean_env(envs)
   sh "rake deep_clean"
 end
 
+desc "generate a release tarball"
+task :release => :compile do
```

Comment:
I'd like to merge your changes and test it more before we make the `:release` task public.

Once we're happy with it, we should be able to just flip the switch and add the task to setup.